tests/integration/gateway/gateway_clients/test_streaming.py

```diff
@@ -1,22 +1,22 @@
-import time
-import os
 import asyncio
 import copy
 import multiprocessing
-import pytest
-
-from typing import Dict
+import os
```

Comment:
these changes here are probably isort, but lets try to not change imports on files you are making no other changes?

Reply:
That's strange that it has been caught in the pre-commit then :thinking: I will remove it